test/e2e/snaps/test-snap-error.spec.js

```diff
@@ -52,24 +63,23 @@ describe('Test Snap Error', function () {
         });
 
         // click send inputs on test snap page
-        await driver.waitUntilXWindowHandles(1, 5000, 10000);
+        await driver.waitUntilXWindowHandles(2, 5000, 10000);
         windowHandles = await driver.getAllWindowHandles();
         await driver.switchToWindowWithTitle('Test Snaps', windowHandles);
         await driver.delay(1000);
+
+        // find and click on send error
         await driver.clickElement('#sendError');
 
-        await driver.navigate(PAGES.HOME);
+        // switch back to the extension page
+        await driver.switchToWindow(extensionPage);
+        await driver.delay(1000);
 
-        const error = await driver.findElement(
-          '.home-notification__content-container',
-        );
-        const text = await error.getText();
-        assert.equal(
-          text.includes(
-            "Snap Error: 'random error inside'. Error Code: '-32603'",
-          ),
-          true,
-        );
+        // try to click on the dismiss button and pass test if it works
+        await driver.clickElement({
```

Comment:
Why was this changed to not find the text anymore?

Reply:
It assumes the error is present because the dismiss button is present. 
Also the new error snap doesn't return a "true" to the interface anymore after the new snap was edited by maarten. Is it necessary?

Reply:
fixed this with 9bec385